react-native-in-app-payments-quickstart/app/components/OrderInformationTitleView.tsx

```diff
@@ -24,7 +24,7 @@ OrderInformationTitleView.propTypes = {
   title: PropTypes.string.isRequired,
 };
 
-export default function OrderInformationTitleView({ title }) {
+export default function OrderInformationTitleView({ title }: { title: any }) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.